src/ext/coms_debug.c

```diff
@@ -47,12 +47,12 @@ uint32_t ddtrace_coms_test_writers(void) {
 }
 
 uint32_t ddtrace_coms_test_consumer(void) {
-    if (!ddtrace_coms_rotate_stack(true)) {
+    if (!ddtrace_coms_rotate_stack(true, atomic_load(&ddtrace_coms_globals.stack_size))) {
         printf("error rotating stacks");
```

Comment:
I know this is old code, since this is a new code path, we probably want to replace all `printf()`'s with `ddtrace_log_debugf()` / `ddtrace_log_debug()` to ensure errors with the background sender do not result in 500 errors.